ravenframework/Simulation.py

```diff
@@ -656,6 +658,8 @@ def __readRunInfo(self, xmlNode, runInfoSkip, xmlFilename):
         self.runInfoDict['internalParallel'] = utils.interpretBoolean(element.text)
```

Comment:
Do you think we should raise a deprecation message here for internalParallel? Or the same message you put in the documentation that suggests using `<parallelMethod>`

Reply:
Hm, after creating the pull request, I realized that internalParallel does have one feature that parallelMethod is missing.  I did not add a "Just Parallelize it, I don't care how" ability, which is what internalParallel = True currently means. So maybe I should add a parallelMethod=Distributed or something like that, which is an exact match of internalParallel = True before deprecating it.